pkg/storage/ec/client.go

```diff
@@ -74,8 +74,8 @@ func (ec *ecClient) Put(ctx context.Context, nodes []*proto.Node, rs eestream.Re
 	if !unique(nodes) {
 		return Error.New("duplicated nodes are not allowed")
 	}
-	padded := eestream.PadReader(ioutil.NopCloser(data), rs.DecodedBlockSize())
-	readers, err := eestream.EncodeReader(ctx, padded, rs, ec.mbm)
+	
+	readers, err := eestream.EncodeReader(ctx, data, rs, ec.mbm)
```

Comment:
I'd suggest to not remove the padding from ECClient for now. The encryption scheme will be configurable and I imagine one of the options available will be "no encryption". So we will still need to pad the data when no encryption is chosen.

A possible optimization would be to check if the data needs padding before wrapping the reader with PadReader, but this could be done in a separate PR.